src/panels/lovelace/editor/lovelace-editor/hui-dialog-edit-lovelace.ts

```diff
@@ -0,0 +1,39 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import { Lovelace } from "../../types";
+
+import { HomeAssistant } from "../../../../types";
+import "./hui-edit-lovelace";
+
+export class HuiDialogEditLovelace extends LitElement {
```

Comment:
This class does literally nothing. It passes all received properties and function calls to the element it renders

Reply:
That's what I meant, it has a lot of stuff now that is not needed for how simple it is now....